hs/Database/CourseQueries.hs

```diff
@@ -120,6 +123,18 @@ buildSession lecs tuts =
                           (map entityVal tuts)
 
 -- ** Other queries
+-- | Gets Shape, Text and Path elements and returns as JSONs
+getJSONs :: Int64 -> IO(Response)
```

Comment:
Change the name. Should involve "Graph", and JSON isn't plural.

Put a space after IO.